go.mod

```diff
@@ -98,8 +98,8 @@ require (
 	k8c.io/kubelb v1.1.2
 	k8c.io/kubeone v1.7.3
 	k8c.io/kubermatic/sdk/v2 v2.0.0-00010101000000-000000000000
-	k8c.io/machine-controller/sdk v1.62.0
-	k8c.io/operating-system-manager v1.7.1-0.20250704091613-71af543cb934
+	k8c.io/machine-controller/sdk v1.63.0
+	k8c.io/operating-system-manager v1.7.5
```

Comment:
As far as I can say, this might be an unintended downgrade (commit https://github.com/kubermatic/operating-system-manager/commit/71af543cb934 is on `main` of OSM only).

Reply:
Bumping OSM  since the commit change is available to the release branch v1.7 as well.